File: easygallery-core/src/main/java/za/co/bwmuller/easygallerycore/data/custom/AlbumLoaderCallback.java
package za.co.bwmuller.easygallerycore.data.custom;

import java.util.ArrayList;

import za.co.bwmuller.easygallerycore.model.Album;

/**
 * Created by Bernhard Müller on 8/24/2017.
 */

public interface AlbumLoaderCallback {
    ArrayList<Album> prefixAlbums();

    ArrayList<Album> postfixAlbums();
}

File: easygallery-core/src/main/java/za/co/bwmuller/easygallerycore/data/custom/AlbumLoaderCallback.kt
package za.co.bwmuller.easygallerycore.data.custom

import za.co.bwmuller.easygallerycore.model.Album
import java.util.ArrayList

/**
 * Created by Bernhard Müller on 8/24/2017.
 */
interface AlbumLoaderCallback {

    fun prefixAlbums(): ArrayList<Album>?
    fun postfixAlbums(): ArrayList<Album>?
}